aten/tools/run_tests.sh

```diff
@@ -3,75 +3,78 @@ set -x
 set -e
 
 VALGRIND_SUP="${PWD}/`dirname $0`/valgrind.sup"
-pushd $1
+export CPP_TESTS_DIR=$1
 
 VALGRIND=${VALGRIND:=ON}
-./basic
-./atest
-./scalar_test
-./broadcast_test
-./wrapdim_test
-./apply_utils_test
-./dlconvertor_test
-./native_test
-./scalar_tensor_test
+python test/run_test.py --cpp --verbose -i \
+  cpp/basic \
+  cpp/atest \
+  cpp/scalar_test \
+  cpp/broadcast_test \
+  cpp/wrapdim_test \
+  cpp/apply_utils_test \
+  cpp/dlconvertor_test \
+  cpp/native_test \
+  cpp/scalar_tensor_test \
+  cpp/undefined_tensor_test \
+  cpp/extension_backend_test \
+  cpp/lazy_tensor_test \
+  cpp/tensor_iterator_test \
+  cpp/Dimname_test \
+  cpp/Dict_test \
+  cpp/NamedTensor_test \
+  cpp/cpu_generator_test \
+  cpp/legacy_vmap_test \
+  cpp/operators_test
+
 if [[ -x ./tensor_interop_test ]]; then
-  ./tensor_interop_test
+  python test/run_test.py --cpp --verbose -i cpp/tensor_interop_test
 fi
-./undefined_tensor_test
-./extension_backend_test
-./lazy_tensor_test
-./tensor_iterator_test
-./Dimname_test
-./Dict_test
-./NamedTensor_test
-./cpu_generator_test
-./legacy_vmap_test
-./operators_test
 if [[ -x ./cudnn_test ]]; then
-  ./cudnn_test
+  python test/run_test.py --cpp --verbose -i cpp/cudnn_test
 fi
 if [[ -x ./cuda_generator_test ]]; then
-  ./cuda_generator_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_generator_test
 fi
 if [[ -x ./apply_test ]]; then
-  ./apply_test
+  python test/run_test.py --cpp --verbose -i cpp/apply_test
 fi
 if [[ -x ./stream_test ]]; then
-  ./stream_test
+  python test/run_test.py --cpp --verbose -i cpp/stream_test
 fi
 if [[ -x ./cuda_half_test ]]; then
-  ./cuda_half_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_half_test
 fi
 if [[ -x ./cuda_vectorized_test ]]; then
-  ./cuda_vectorized_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_vectorized_test
 fi
 if [[ -x ./cuda_distributions_test ]]; then
-  ./cuda_distributions_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_distributions_test
 fi
 if [[ -x ./cuda_optional_test ]]; then
-  ./cuda_optional_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_optional_test
 fi
 if [[ -x ./cuda_tensor_interop_test ]]; then
-  ./cuda_tensor_interop_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_tensor_interop_test
 fi
 if [[ -x ./cuda_complex_test ]]; then
-  ./cuda_complex_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_complex_test
 fi
 if [[ -x ./cuda_complex_math_test ]]; then
-  ./cuda_complex_math_test
+  python test/run_test.py --cpp --verbose -i cpp/cuda_complex_math_test
```

Comment:
Why make these verbose? Does this turn them into "as verbose as normal python tests" mode, or do cpp tests need to be extra verbose for some reason?